kolibri_explore_plugin/assets/src/customApps.js

```diff
@@ -295,6 +295,7 @@ export const ChannelCardIcons = {
   '08897e003ea9489eb3d86fc94ba08c21': 'support-ukraine.png',
   '608fd926be60462aba2feccba6c050c9': 'harper-collins.png',
   '6da97a60cbd44e918137827bd5425efb': 'code-org.png',
+  '8fa678af1dd05329bf3218c549b84996': 'phet-interactive-simulations.png', // phet spanish
```

Comment:
Don't we want the same customization for the Part English as well?

Reply:
Yes, for some reason I could not figure out the English channel has the right channel icon. I could make it explicit, I guess?

Reply:
Ok, IIUC we do NOT need the same customization for Phet English. In that case, this LGTM.